openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -79,7 +79,7 @@ W3C Verifiable Presentation:
 :  A Verifiable Presentations compliant to the [@VC_DATA] specification.
 
 Credential Issuer:
-:  An entity that issues Verifiable Credentials. Also called Issuer. In the context of this specification, the Credential Issuer acts as an OAuth 2.0 Authorization Server (see [@!RFC6749]).
+:  An entity that issues Verifiable Credentials. Also called Issuer. In the context of this specification, the Credential Issuer acts as an OAuth 2.0 Resource Server (see [@!RFC6749]).
```

Comment:
Not really sure about this one, but I think referencing an AS here makes sense. What do you think?
```suggestion
:  An entity that issues Verifiable Credentials. Also called Issuer. In the context of this specification, the Credential Issuer acts as an OAuth 2.0 Resource Server (see [@!RFC6749]) and utilizes an OAuth 2.0 Authorization Server to manage access.
```

Reply:
Honestly, the intent of this PR was just to fix the one error and I think it should stay with that.